modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountGroupLocalServiceImpl.java

```diff
@@ -54,6 +66,26 @@ public AccountGroup addAccountGroup(
 		return accountGroupPersistence.update(accountGroup);
 	}
 
+	@Override
+	public BaseModelSearchResult<AccountGroup> search(
+		long companyId, String keywords, int start, int end,
+		OrderByComparator obc) {
+
+		DynamicQuery accountGroupDynamicQuery = _getAccountGroupDynamicQuery(
+			companyId, keywords, obc);
+
+		if (accountGroupDynamicQuery == null) {
+			return new BaseModelSearchResult<>(
+				Collections.<AccountGroup>emptyList(), 0);
+		}
```

Comment:
This check is not necessary. `_getAccountGroupDynamicQuery` will never return null.